lms/djangoapps/static_template_view/views.py

```diff
@@ -50,7 +52,12 @@ def render(request, template):
         # This is necessary for the dialog presented with the TOS in /register
         if template == 'honor.html':
             context['allow_iframing'] = True
-        return render_to_response('static_templates/' + template, context, content_type=content_type)
+        # Format Examples: static_template_about_header
+        configuration_base = 'static_template_' + template.replace('.html', '').replace('-', '_')
+        context['page_header'] = configuration_helpers.get_value(configuration_base + '_header')
+        context['page_content'] = SafeUnicode(configuration_helpers.get_value(configuration_base + '_content'))
```

Comment:
Wouldn't we want to allow HTML markup in the header as well?  At least HTML entities like `&auml;` should be allowed, so we should add `SafeUnicode` there as well.

Reply:
I agree. Added.

Reply:
After looking at and trying a few different ways to collapse these down, I haven't been able to find something that both works, and meaningfully reduces the duplication. It just added more code, another new template, and about ~40% less templates.

One other possibility is deleting all of the more "generic" template files and just creating a single generic template that you pass in the Title and Header in into the view.

But I felt that was changing the functionality a bit too far as changing the SiteConfiguration would be the only way.